src/use_case/delete_folder/DeleteFolderUserDataAccessInterface.java

```diff
@@ -4,4 +4,6 @@
 
 public interface DeleteFolderUserDataAccessInterface {
     String deleteFolder(String folderName);
+
+    boolean existsByName(String identifier);
```

Comment:
why do you add existsByName in DeteleFolder? when you press the button on the folder, the folder should exist. there is no need to add this method for checking

Reply:
this can delete, i will update this